CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # @financialforcedev/eslint-config
 
+## 4.0.0
```

Comment:
Sorry I don't get why version 4.0.0 is in ChangeLog 3?

Reply:
It's been renamed to CHANGELOG.md, we now just put everything into one file 👍